src/components/TransactionDisplay.tsx

```diff
@@ -14,47 +14,61 @@
 // You should have received a copy of the GNU General Public License
 // along with Parity Bridges UI.  If not, see <http://www.gnu.org/licenses/>.
 
-import { Card, CardContent, CardHeader, Container } from '@material-ui/core';
+import { Box, Card, makeStyles, Typography } from '@material-ui/core';
 import React from 'react';
-import styled from 'styled-components';
 
-import { Step, TransactionStatusEnum, TransanctionStatus } from '../types/transactionTypes';
+import { Step, TransanctionStatus } from '../types/transactionTypes';
+import { ButtonSwitchMode } from './Buttons';
+import { IconTxStatus } from './Icons';
 
 interface Props {
-  className?: string;
   steps: Array<Step>;
   transaction: TransanctionStatus;
 }
 
-const TransactionDisplay = ({ transaction, steps, className }: Props) => {
-  if (!steps.length) {
-    return null;
+const useStyles = makeStyles((theme) => ({
+  card: {
+    '& p': {
+      ...theme.typography.body2
+    },
+    '& svg': {
+      marginBottom: '-0.2em',
+      fontSize: '1.2em',
+      marginRight: theme.spacing()
+    },
+    '& .header': {
+      fontWeight: 500
+    }
   }
+}));
+
+export const TransactionDisplay = ({ transaction, steps }: Props) => {
+  const classes = useStyles();
+  if (!steps.length) return null;
```

Comment:
You should always return a component and never a null. 
It is better to make the length step BEFORE rendering the component. That means outside the TransactionDisplay - before it is called... something like: 
```
!steps.length && <TransactionDisplay transaction={transaction} steps={steps} />
```

Reply:
i'm not agree with this, returning null is quite used and actually is documented 
https://reactjs.org/docs/conditional-rendering.html#preventing-component-from-rendering

Reply:
ok, I'm not correcting unless it is decided otherwise

Reply:
Just think of it this way:
Why to render (and go through the lifecycle) of a component as soon as you want to return null? - just do not render it.

Documentation you sent talks about "rare" situations (I would use it only if some internal option of the component was needed to hide it - some state change etc etc).

Reply:
i see what you mean, but anyway lifecycle components is not a such expensive process to avoid.

Reply:
I haven't corrected this for now, will wait for consensus from memory mgmt gurus! Pls approve

Reply:
will be resolved in https://github.com/paritytech/parity-bridges-ui/issues/108